### DISCUSSION TITLE
Sidebar "Pass Through" documentation corrections.

### DISCUSSION THREAD

**Discussion by charithAmila** [primefaces/community#1332] [PrimeReact] [answered]
Option number three (3), five (5), and (6) under the Sidebar PT Options section is not match with the Wireframe.

1. Option 3 is not "closeButtonIcon" 🤔.
2. Option 4 should be  "icons".
3. Option 5 should be "closeIcon".
4. Option 6 should be "content".

<img width="414" alt="Screen Shot 2024-02-29 at 3 45 20 PM" src="https://github.com/primefaces/community/assets/10193181/30004eca-c248-47f4-a4b1-977128a1f7ca">

<img width="836" alt="Screen Shot 2024-02-29 at 3 49 00 PM" src="https://github.com/primefaces/community/assets/10193181/267e2dfc-8914-466d-815f-66b8cf327ad1">

I would love to raise a PR for this issue.

**Comment by melloware** [comment, marked as answer]
Thanks issue raised: https://github.com/primefaces/primereact/issues/6068
